cpp/src/rolling/rolling_detail.hpp

```diff
@@ -101,7 +101,7 @@ struct rolling_store_output_functor<_T, true> {
   template <typename T = _T, std::enable_if_t<cudf::is_timestamp<T>()> * = nullptr>
   CUDA_HOST_DEVICE_CALLABLE void operator()(T &out, T &val, size_type count)
   {
-    out = val.time_since_epoch() / count;
+    out = static_cast<T>(val.time_since_epoch() / count);
```

Comment:
Mean of timestamp types should be disallowed now that we have a duration type. Is this addressed in a different PR?

Reply:
that is going to be addressed in a different pr - presumably #5419. @karthikeyann was going to work on the duration support to rolling/groupby etc.

Reply:
PR #5319 for disabling MEAN, SUM of timestamp types.